packet_pool.go

```diff
@@ -37,35 +88,17 @@ func (b *packetPool) add(p *Packet) (ps []*Packet) {
 	defer b.m.Unlock()
 
 	// Init buffer
-	var mps []*Packet
+	var acc *packetAccumulator
```

Comment:
I'd rather simplify this with :

```go
// Make sure accumulator exists
if _, ok := b.b[p.Header.PID]; !ok {
	b.b[p.Header.PID] = newPacketAccumulator(b.dmx, p.Header.PID)
}

// Add to the accumulator
return b.b[p.Header.PID].add(p)
```

What do you think?

Reply:
Agreed. I kept the old structure but it's not necessary here. Will change.